cookbooks/bcpc-hadoop/attributes/default.rb

```diff
@@ -172,18 +148,16 @@
   mapred
   httpfs
 )
+default[:bcpc][:hadoop][:os][:group][:hdfs][:members]=['hdfs']
+default[:bcpc][:hadoop][:os][:group][:mapred][:members]=['yarn']
```

Comment:
Style:

1. Strings instead of symbols
2. Space between attribute, operator and value

Reply:
Fixed whole file's attribute access and corrected merge issue.